tests/functionaltests/Tests/NSURLSession.mm

```diff
@@ -68,6 +69,48 @@ - (NSURLSession*)createSession {
 }
 
 //
+// Property getters
+//
+// Note: Since we're using @synchronized(self), the atomic annotation isn't strong enough for
```

Comment:
Incidentally, a mutable array property returned through synchronized locking and then queried is not itself concurrency-safe, even if the mutation is performed synchronized as well... however, since we're guaranteeing that mutation _should be_ done (with the boolean condition), it's "safer" than what we had before.